apps/monsteer_spike_receiver.cpp

```diff
@@ -54,20 +54,20 @@ class SpikeReceiver
 
     void run()
     {
-        float time = 0;
-        while( _reader.waitUntil( time ))
-        {
-            const brion::Spikes& spikes = _reader.getSpikes();
-            BOOST_FOREACH( const brion::Spike& spike, spikes )
-                std::cout << spike.first << " " << spike.second << std::endl;
+//        float time = 0;
+//        while( _reader.waitUntil( time ))
+//        {
+//            const brion::Spikes& spikes = _reader.getSpikes();
+//            BOOST_FOREACH( const brion::Spike& spike, spikes )
```

Comment:
This code was commented. The implementation is the same as Brion/apps/spikeConverter
May be that it should be removed.